packages/block-library/src/comments-query-loop/editor.scss

```diff
@@ -0,0 +1,3 @@
+.block-library-comments-toolbar__popover .components-popover__content {
+	min-width: 230px;
+}
```

Comment:
While working on #40522, @DAreRodz, @michalczaplinski and I found that this file doesn't seem to be used.

It seems that it was introduced to fix the issue mentioned [here](https://github.com/WordPress/gutenberg/pull/37297#issuecomment-995535517), but it appears the relevant settings were [later moved to the inspector controls](https://github.com/WordPress/gutenberg/pull/37297#issuecomment-998101414) in the sidebar, making the styling obsolete.

Just documenting this here for posterity 😄